network/src/commonMain/kotlin/com/wire/kalium/network/api/v2/authenticated/KeyPackageApiV2.kt

```diff
@@ -0,0 +1,51 @@
+package com.wire.kalium.network.api.v2.authenticated
+
+import com.wire.kalium.network.AuthenticatedNetworkClient
+import com.wire.kalium.network.api.base.authenticated.keypackage.ClaimedKeyPackageList
+import com.wire.kalium.network.api.base.authenticated.keypackage.KeyPackage
+import com.wire.kalium.network.api.base.authenticated.keypackage.KeyPackageApi
+import com.wire.kalium.network.api.base.authenticated.keypackage.KeyPackageCountDTO
+import com.wire.kalium.network.api.base.authenticated.keypackage.KeyPackageList
+import com.wire.kalium.network.api.v0.authenticated.KeyPackageApiV0
+import com.wire.kalium.network.utils.NetworkResponse
+import com.wire.kalium.network.utils.wrapKaliumResponse
+import io.ktor.client.request.get
+import io.ktor.client.request.parameter
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+
+internal class KeyPackageApiV2 internal constructor(
+    authenticatedNetworkClient: AuthenticatedNetworkClient
+) : KeyPackageApiV0(authenticatedNetworkClient) {
```

Comment:
Why do we override everything? Because we'll delete `KeyPackageApiV0`?

Reply:
i copied the implementation form v0 to v2 because after adding a way to use v2 (there isn't yet) all MLS endpoints in v0 will return not supported instead of deleting `KeyPackageApiV0` this way logic still have one abstraction layer to use